pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[build-system]
+requires = ["setuptools", "wheel", "versioneer"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "MembraneCurvature"
+version = "1.0.0"
+license = {file = "LICENSE"}
+description = "MDAnalysis tool to calculate membrane curvature from MD simulations."
+authors = [
+    {name = "Estefania Barreto-Ojeda", email = "estefania@ojeda-e.com"},
+]
+maintainers = [
+    {name = 'MDAnalysis Core Developers', email = 'mdanalysis@numfocus.org'}
```

Comment:
Pinging @orbeckst due to previous discussions on the topic - I'm not 100% sure if the coredevs can take on full maintenance right now (maybe they can, but I could not make that decision for them).

Reply:
We can't promise maintaining the kit. Therefore we need to find another name here. 

In the absence of a maintainer, I suggest putting @ojeda-e back in there. Then there's at lease contact information and it's the historical status. I don't know what the best way is to indicate "looking for maintainer".

Reply:
Thanks @orbeckst.
I will add my name as maintainer until there is a candidate.